.github/prettierignore

```diff
@@ -0,0 +1,29 @@
+ISSUE_TEMPLATE
+PULL_REQUEST_TEMPLATE
+workflows
+dependabot.yml
+.gitpod.yml
+# .netlify
```

Comment:
Shouldn't `. netlify` also be ignored?

Reply:
(I understand that this file's content is preliminary and that the main purpose of this PR is to setup the workflow. So feel free to ignore this and the next comment about tweaking the lines in this file. Though I still think that it should be moved to the top-level and renamed :))